src/ohri-form.component.tsx

```diff
@@ -169,6 +164,30 @@ const OHRIForm: React.FC<OHRIFormProps> = ({
     reportError(patientError, t);
   }, [patientError, t]);
 
+  const WarningModal = () => {
```

Comment:
This modal gets shown when the user enters data into a form but then tries to navigate away from the form without first saving it.

Reply:
This component will get created on every render cycle of the the host component. Can we wrap it in a 'useCallback' or even better migrate it to its own file?